tests/std/tests/P0645R10_text_formatting_parsing/test.cpp

```diff
@@ -156,6 +168,33 @@ constexpr bool test_parse_arg_id() {
     return true;
 }
 
+template <typename CharT>
+constexpr bool test_parse_precision() {

```

Comment:
Should there be test coverage for the exception cases?

Reply:
there should definitely be tests for both exception paths, however I'm not sure about tests that check that the correct one was thrown, we probably want to have multiple subclasses of format_error at some point.

Reply:
Is this recorded somewhere (project note, todo comment, etc?). I don't want us to forget this.

Reply:
added card to the todo section of the project